hydroshare/settings.py

```diff
@@ -701,6 +701,9 @@
 
 HSWS_ACTIVATED = False
 
+# HydroShare THREDDS Data Server URL
+THREDDS_SERVER_URL = 'https://thredds.hydroshare.org/thredds/'
```

Comment:
This is only used for display so I think it should be fine in settings.py.  The only concern I sort of have is it might lead to some confusion on beta.

Reply:
Could you add `HSWS_GEOSERVER_URL = "https://geoserver.hydroshare.org/geoserver"` to settings.py while you're in here as well?  I've seen logs on some dev machines where that key hasn't been added to local_settings.py and this would avoid the issue entirely.